Yona/Yona/Others/SMSValidPasscodeLogin/SMSValidation/ConfirmMobileValidationVC.swift

```diff
@@ -65,8 +65,23 @@ class ConfirmMobileValidationVC: ValidationMasterView {
 }
 
 extension ConfirmMobileValidationVC: CodeInputViewDelegate {
+    
+    func handleNavigtaionFromProfileView() {
```

Comment:
Typo: ``handleNavigtaionFromProfileView`` should be ``handleNavigationFromProfileView``.

Reply:
Updated.